.github/workflows/build.yml

```
@@ -329,6 +329,21 @@ jobs:
       run: signtool sign /f $Env:WINDOWS_CODESIGN_CERTIFICATE_PATH /p $Env:WINDOWS_CODESIGN_CERTIFICATE_PASSWORD *.msi
       working-directory: build
 
+    - name: "Prepare for deployment"
+      # Copy the desired directory structure to the deploy/ directory. This
+      # also generates metadata for file artifact and write it to the job
+      # output using the artifacts_slug value.
+      id: prepare_deploy
+      if: github.event_name == 'push'
+      shell: bash
+      run: >
+        python3 tools/deploy.py prepare-deployment
+        --slug '${{ matrix.artifacts_slug }}'
+        --output-dir 'deploy/'
+        --dest-path 'builds/{branch}/${{ runner.os }}/{filename}'
```

For release tags, how about doing
```suggestion
        --dest-path '{ tag }/${{ runner.os }}/{filename}'
```

---

I was planning to improve the naming in a follow-up PR, but this one just keeps everything as before.